en/docs/learn/ai/agents/introduction-to-chat-agents.md

```diff
@@ -1,4 +1,4 @@
-# Build an AI Agent to Handle Mathematical Queries
+# Introduction to Chat Agents
```

Comment:
Can you fix the below titles. We might have to reword as those are not good titles.

<img width="1437" alt="image" src="https://github.com/user-attachments/assets/a6c348a5-e819-4b9f-a9a4-cf3e4b30e56a" />